packages/react-magma-dom/src/components/Pagination/SimplePagination.tsx

```diff
@@ -119,6 +118,8 @@ export const SimplePagination = React.forwardRef<
         }`;
   }
 
+  const pageAriaLabel = `Page number ${selectedPage} ${paginationLabel()} selected`;
```

Comment:
Just tested and this is read by VoiceOver **beautifully**! The only thing is that we should i18n `Page number` and `selected` in case this is ever used in another language.